notebooks/data-collection.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
<p>Can we add a title/headline for this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-data-science-community/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>